Backend/README.md

```diff
@@ -118,6 +118,13 @@ If you want to access the database, you are most likely prompted for a password.
       "grade": double,
       "graduation_date": String
     }
+  ],
+  "keyCompetencies": [
+    {
+      "category: KeyCompetenciesCategory,
```

Comment:
fine **Syntax** error by "category: KeyCompetenciesCategory,

Reply:
You mean the missing quote after `"category`?

Reply:
yeah, and KeyCompetenciesCategory should be declared